website/docs/integrator-section/webapp-tutorial/revoke-document-store-webapp.md

```diff
@@ -1,31 +1,43 @@
 ---
 id: revoke-document-store-webapp
-title: Revoke Document with Web app
-sidebar_label: Revoke Document Store
+title: Revoking Document with Webapp
+sidebar_label: Revoking Document Store
```

Comment:
The existing sidebar labels do not make sense to me. 
`Deploy Document Store` is correct but you cannot issue or revoke document stores. 

Instead it should be Issue documents and Revoke documents

Reply:
updated.